vitest.setup.ts

```
@@ -0,0 +1,27 @@
+import '@testing-library/jest-dom'
+import { vi } from 'vitest';
+import * as matchers from 'jest-extended';
+
+// Setup jest-extended for additional matchers (e.g., toHaveBeenCalledBefore, ...etc)
+expect.extend(matchers); 
+
+// Mock Next.js components/functions
+vi.mock('next/server', () => ({
+  NextResponse: {
+    json: (data: any, init?: ResponseInit) => 
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)